api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java

```diff
@@ -38,7 +38,7 @@ public abstract class AbstractIntegrationTest {
   private static final boolean IS_ARM =
       System.getProperty("os.arch").contains("arm") || System.getProperty("os.arch").contains("aarch64");
 
-  private static final String CONFLUENT_PLATFORM_VERSION = IS_ARM ? "7.2.1.arm64" : "7.2.1";
+  private static final String CONFLUENT_PLATFORM_VERSION = IS_ARM ? "7.8.0.arm64" : "7.8.0";
```

Comment:
hi there
as you noticed you have some failed tests in you MR. They fail because you bumbed confluentinc/cp-kafka version here.
I tested it and here is what I figured out:
1. Test fails randomly. Sometimes it passes, sometimes not. 
2. Such problem starts from '7.5.0' confluentinc/cp-kafka version and further

You can check release notes from: 
https://docs.confluent.io/platform/7.5/release-notes/index.html

I assume the problem related to Kafka Connect initialization. I suggest you to use here 7.4.0 version as last that works and try to ping Confluent guys with this problem.

Reply:
@DimaVilda @yeikel PTAL at this: https://github.com/kafbat/kafka-ui/pull/573#issuecomment-2464974900, might be related.

Reply:
aha, I see
I don't like the workaround, actually and I would vote it will be resolved on Confluent's side, up to you @Haarolean 
In this MR i'd suggest to use stable kafka image version and don't bother with it

Reply:
The comment links to https://issues.apache.org/jira/browse/KAFKA-14455 but that issue was "resolved" and the fix should be included in 7.8

Reply:
Looking at the comment and the issue linked 

> The image upgrade introduced a bug in the tests because some kafka connect errors that were previously processed asynchronously are now rising up to the POST|PUT /connectors responses. See the [KAFKA-14455](https://issues.apache.org/jira/browse/KAFKA-14455) issue.
> 
> Specifically it now randomly produces HTTP 500 errors because Kafka Connect often falls into the "re balancing" state when we create/delete connectors in a row.

I confirmed that this issue started in `7.5.x`. I could not replicate this issue in any version of `7.4.x`.

The reason this issue started is due to the linked JIRA ticket: https://issues.apache.org/jira/browse/KAFKA-14455, which addressed an existing problem in error feedback for Connect. 

As described in the ticket, this was a case of poor user experience where the connector endpoint previously returned `200` even when the call failed asynchronously internally not saving the actual user change and producing a misleading output. With the new change, when a re-balance is in progress, the API returns 500 instead as expected

With this context, I do not believe this is a source of concern. In fact, the reason the tests passed earlier was due to the bad user experience noted in the original issue, where errors were not correctly propagated to users.

I think that moving forward, instead of reporting this upstream, we should update the tests to accommodate the expected behavior, either as part of this pull request or by incorporating the changes implemented by @Dugong42 in his other pull request.

Reply:
Based on the tests I conducted, it seems that most failures occur due to the `@setUp()` method, which attempts to perform another mutation (the connector creation) before the backend has finished processing the previous operation. Essentially, this creates a situation where tests indirectly conflict with each other’s state.

What I am currently trying to determine is what exactly connect is doing that necessitates waiting before proceeding with the next test.

One obvious alternative is to ensure that each test uses its own instance of connect ensuring that we create a clean instance of the dependencies with each test and that no state is shared among tests. The downside of doing that is that the startup of each test will be considerably slower.

Reply:
I updated the code to [retry on Rebalance ](https://github.com/yeikel/kafka-ui/blob/5c1a7e503e8b4733e505dcefcf00fb1a7c96d6ed/api/src/main/java/io/kafbat/ui/client/RetryingKafkaConnectClient.java#L55-L66)and the tests seem more stable right now. That's the solution I believe we should deploy